packages/mdc-menu-surface/component.ts

```diff
@@ -93,7 +93,7 @@ export class MDCMenuSurface extends MDCComponent<MDCMenuSurfaceFoundation> {
   }
 
   /**
-   * Removes the menu-surface from it's current location and appends it to the
+   * Removes the menu-surface from its current location and appends it to the
```

Comment:
Can we remove this method from menu surface since no longer used?

Reply:
Will do in a follow-up PR, since this one is already huge.